src/Nncase.Tests/Rewrite/RewriteBase.cs

```diff
@@ -40,6 +40,13 @@ public interface IRewriteCase
     /// Gets the eval inputs dict.
     /// </summary>
     Dictionary<Var, IValue> FeedDict { get; }
+
+    /// <summary>
+    /// check rewrite post callback.
+    /// </summary>
+    /// <param name="post"></param>
+    /// <returns></returns>
+    bool CheckPostCallBack(Function post) => true;
```

Comment:
What's the usage of `CheckPostCallBack`?

Reply:
Sometimes need check the post function structure.

Reply:
Should naming `ChecksPostCallBack`.